src/tests/unit/auto/get_device_list.cpp

```diff
@@ -95,20 +94,21 @@ TEST_P(GetDeviceListTest, GetDeviceListTestWithExcludeList) {
 // ConfigParams {devicePriority, expect metaDevices, ifThrowException}
 
 const std::vector<ConfigParams> testConfigs = {
-    //
-    ConfigParams {"CPU,GPU,VPUX",
-        "CPU,GPU,VPUX"},
-    ConfigParams {"VPUX,GPU,CPU,-GPU.0",
-        "VPUX,GPU.1,CPU"},
-    ConfigParams {"-GPU.0,GPU,CPU",
-        "GPU.1,CPU"},
-    ConfigParams {"-GPU.0,GPU",
-        "GPU.1"},
-    ConfigParams {"-GPU.0", "CPU,GPU.1,VPUX"},
-    ConfigParams {"-GPU.0,-GPU.1", "CPU,VPUX"},
-    ConfigParams {"-GPU.0,-CPU", "GPU.1,VPUX"}
-};
-
+    ConfigParams{" ", " "},
```

Comment:
Do we need add test for:
ConfigParams{"", " "},

Reply:
I think this is necessary for the test case of inputting device priorities like "AUTO: ".